test/rules/src/https_everywhere_checker/rules.py

```diff
@@ -179,6 +179,37 @@ def _determineTestApplication(self):
 					self.test_application_problems.append("%s: No rule or exclusion applies to test URL %s" % (
 						self.filename, test.url))
 				self.determine_test_application_run = True
+
+			for test in self.tests:
+				urlParts = urlparse(test.url)
+				hostname = urlParts.hostname
+				ok = False
```

Comment:
Please be more descriptive in this variable name.  Also, for the first case, we can shorten it up:

```python
ok = hostname in self.targets
```